src/styles/styles.js

```diff
@@ -812,13 +812,19 @@ const styles = {
         textAlignVertical: 'center',
     },
 
-    textInputPrefix: {
+    textInputPrefixWrapper: {
```

Comment:
Any specific reason we added this style? and it wasn't a part of the proposal?

Reply:
Yes, I thought to move the View related style to the wrapper which isn't fit to the Text element. On web, `Text` component renders `div` to display text which eventually works with `alignItems` to center as it has children but for native won't be the same case. So usually we do provide these styles to the parent `View`.
It wasn't a part of the proposal as I missed the native part test.